mm2src/mm2_main/src/lp_swap.rs

```diff
@@ -236,10 +237,12 @@ pub fn broadcast_p2p_tx_msg(ctx: &MmArc, topic: String, msg: &TransactionEnum, p
     broadcast_p2p_msg(ctx, vec![topic], encoded_msg, from);
 }
 
-pub async fn process_msg(ctx: MmArc, topic: &str, msg: &[u8]) {
+pub async fn process_msg(ctx: MmArc, topic: &str, msg: &[u8]) -> P2PRequestResult<()> {
```

Comment:
No sense to return any kind of error if it's not processed there

Reply:
It is used https://github.com/KomodoPlatform/atomicDEX-API/blob/e520117a055f920b877ae701c679b9ffabc3b2ee/mm2src/mm2_main/src/lp_swap.rs#L240-L244

Reply:
The place where `process_mng` called. 
`MmError` is not utilized and the fact is the case of impossibility to construct Uuid is not logged